<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameTest</class>
 <widget class="QFrame" name="FrameTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_SLanguageManager">
     <property name="text">
      <string>SLanguageManager</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_SLogger">
     <property name="text">
      <string>SLogger</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_SHttp">
     <property name="text">
      <string>SHttp</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
